crates/ruff_dev/src/generate_ty_options.rs

```diff
@@ -208,19 +222,27 @@ fn format_header(
         .chain(scope)
         .join(".");
 
+    // Some examples show headers which would show up in `pyproject.toml`,
+    if matches!(configuration, ConfigurationFile::TyToml) {
+        if example.contains("tool.ty") {
+            return (String::new(), example.replace("tool.ty.", ""));
+        }
+    }
+
```

Comment:
I don't see any changes if I comment this out. Can you say more why this is needed.